experiments_main.cpp

```diff
@@ -201,8 +214,62 @@ class ExperimentRunner {
         generate_sequences();
         std::cout << "Computing edit distances ... ";
         compute_edit_distance();
-        apply_tuple([&](auto &algorithm, auto &dist) { run_sketch_algorithm(&algorithm, &dist); },
-                    algorithms, dists);
+        apply_tuple(
+                [&](auto &algorithm, auto &dist) {
+                    std::cout << "Running " << algorithm.name << std::endl;
+
+                    // Run the algorithms FLAGS_reruns times, storing the distances and Spearman
+                    // coefficient computed in each run. The average and standard deviation of the
+                    // Spearman coefficients is reported, as well as the Spearman coefficient
+                    // obtained from using the median and average of the distances of all runs.
+                    vector<double> spearman_coefficients(FLAGS_reruns);
```

Comment:
should be `std::vector<>`. How come `vector` works?

Reply:
It was you :P fixed now
https://github.com/ratschlab/Project2020-seq-tensor-sketching/blame/master/util/spearman.hpp#L7